syntax = "proto3";

package syft.core.auth;

import "proto/core/common/common_object.proto";
import "proto/core/io/address.proto";

message HelloRootRequest {
  string username = 1;
  syft.core.io.Address reply_to = 2;
  syft.core.io.Address address = 3;
  syft.core.common.UID msg_id = 4;
}

message HelloRootResponse {
  string message = 1;
  syft.core.io.Address address = 2;
  syft.core.common.UID msg_id = 3;
}